docker/nginx/nginx.conf

```diff
@@ -24,7 +24,7 @@ server {
     # cache when using virtual box.
     sendfile off;
 
-    location / {
+    location /api/ {
```

Comment:
hmm, will this pass the actual URL to the upstream server? I forget how it works. If you saw it working then I believe you :)